src/status_im/contexts/wallet/swap/select_asset_to_pay/view.cljs

```diff
@@ -26,7 +26,7 @@
                          (let [pay-token-symbol (:symbol token)
                                asset-to-receive (if (= pay-token-symbol "SNT") eth-token snt-token)]
                            (rf/dispatch [:wallet.swap/start
-                                         {:asset-to-pay     token
+                                         {:asset-to-pay     {:symbol pay-token-symbol}
```

Comment:
Opening the swap screen crashes with the same 'toLowerCase' error without this change. It's rather difficult to debug though, but I assume it's because we were passing the token with a subset of the networks (the ones with balance), while the swap screen may be expecting all networks there. By passing the symbol only, we're forcing the token to be "rebuilt" with all the networks, but maybe you know what could be the issue and have a better solution.